pages/marina.vue

```diff
@@ -1,18 +1,159 @@
+      </div>
+    </div>
 
-<script>
-export default {
-  data() {
-    return {
-      titulo: 'Dicas de como cuidar do seu cachorro',
-      frase: 'Cachorro independente e sem ansiedade de separação'
+    <div class='grid-container2'>
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/169"/>
+        <h3>{{ lola.titulo }}</h3>
+        <p>{{ lola.frase }}</p>
+        <p>{{ lola.name}}</p>
+      </div>
+    
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/501"/>
+        <h3>{{ miuxa.titulo }}</h3>
+        <p>{{ miuxa.frase }}</p>
+        <p>{{ miuxa.name}}</p>
+      </div>
+    
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/502"/>
+        <h3>{{ thomas.titulo }}</h3>
+        <p>{{ thomas.frase }}</p>
+        <p>{{ thomas.name}}</p>
+      </div>
+    
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/503"/>
+        <h3>{{ tango.titulo }}</h3>
+        <p>{{ tango.frase }}</p>
+        <p>{{tango.name}}</p>
+      </div>
+    </div>
+  
+    <div class='grid-container3'>
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/169"/>
+        <h3>{{ lola.titulo }}</h3>
+        <p>{{ lola.frase }}</p>
+        <p>{{ lola.name}}</p>
+      </div>
+    
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/501"/>
+        <h3>{{ miuxa.titulo }}</h3>
+        <p>{{ miuxa.frase }}</p>
+        <p>{{ miuxa.name}}</p>
+      </div>
+    
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/502"/>
+        <h3>{{ thomas.titulo }}</h3>
+        <p>{{ thomas.frase }}</p>
+        <p>{{ thomas.name}}</p>
+      </div>
+    
+      <div>
+        <img width='300px' height='200px' src="https://placedog.net/503"/>
+        <h3>{{ tango.titulo }}</h3>
+        <p>{{ tango.frase }}</p>
+        <p>{{tango.name}}</p>
+      </div>
+    </div>
+    </div>
+  </div>
+  </template>
+  
+  <script>
+  export default {
+    data() {
+      return {
+        // pesquisar o que é dicionário em javascript
+        lola: {
+          titulo: 'Dicas de como cuidar do seu cachorro',
+          frase: 'Cachorro independente e sem ansiedade de separação',
+          name: 'Lola'
+        },
+        miuxa: {
+          titulo: 'Dicas de como cuidar do seu cachorro',
+          frase: 'Cachorro independente e sem ansiedade de separação',
+          name: 'Miuxa'
+        },
+        thomas: {
+          titulo: 'Dicas de como cuidar do seu cachorro',
+          frase: 'Cachorro independente e sem ansiedade de separação',
+          name: 'Thomas'
+        },
+        tango: {
+          titulo: 'Dicas de como cuidar do seu cachorro',
+          frase: 'Cachorro independente e sem ansiedade de separação',
+          name: 'Tango'
+        }
+      }
     }
   }
-}
-</script>
+  </script>
+  
+  <!-- pesquisar o que é classe em css -->
+  
+  <style>
```

Comment:
vocẽ fez 3 coisas iguais aqui:

```vue
  <style>
  .grid-container1 {
    /* pesquisa o que é display: grid */
    display: grid;
    /* pesquisa o que é grid-template-columns */
    grid-template-columns: auto auto auto auto;
    /* pesquisa o que é background-color */
    text-align: center;
  }
  </style>

  <style>
    .grid-container2 {
      /* pesquisa o que é display: grid */
      display: grid;
      /* pesquisa o que é grid-template-columns */
      grid-template-columns: auto auto auto auto;
      /* pesquisa o que é background-color */
      text-align: center;
    }
    </style>

    <style>
      .grid-container3 {
        /* pesquisa o que é display: grid */
        display: grid;
        /* pesquisa o que é grid-template-columns */
        grid-template-columns: auto auto auto auto;
        /* pesquisa o que é background-color */
        text-align: center;
      }
      </style>
```

veja que a identação do código não está boa. código tem que estár identado
pesquisa no google o que é:
- [ ] Código limpo
- [ ] Código indentado

Aqui você pode deixar o css dentro de apenas 1 <style></style>
Veja que você copiou comentário junto com o código, lembra que eu falei pra você não dar ctrl+c ctrl+v no início, você precisa exercitar escrever cada palavra de código, confia. Os comentários dentro do código você não precisa copiar. `/* comentários */`

Aqui tem 3 tipos de linguagens, então terão 3 tipos de comentários diferentes.

- [ ] Como é comentário em html?
- [ ] Como é comentário em Javascript?
- [ ] Como é comentário em CSS?